### ISSUE TITLE
Options panel

### ISSUE THREAD

**Issue by AlexNisnevich** [AlexNisnevich/untrusted#170]
The menu should have an options panel (alternatively, there can be a separate Options button) that should include the following features:
- color-blind mode / customizable editor color scheme (#84)
- connect GitHub account? (#130)
- import/export game state (#106)
- reset game state
- remap keys (for Colemak, etc. support)
- enable showing coordinates on hover (assuming we don't always enable this) (#27)
- music and sfx volume controls
- others?
